openid-4-verifiable-presentations-1_0.md

```diff
@@ -2218,6 +2189,33 @@ The profile includes the following elements:
 
 ### DCQL Query and Response
 
+This section defines ISO mdoc specific DCQL Query and Response parameters.
+
+#### mdoc specific Parameters in the `meta` parameter in Credential Query {#mdocs_meta_parameter}
```

Comment:
I think we could drop "mdoc specific Parameters in the" here as this whole section is about mdoc